src/app/public/modules/radio/radio.component.ts

```diff
@@ -145,9 +196,22 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._value;
   }
 
+/**
+ * Specifies an icon to display in place of the radio button. To group radio buttons like in
+ * [the demo above](https://developer.blackbaud.com/skyux-forms/docs/radio#demo) place the
+ * `sky-switch-icon-group` class on the direct parent element of the radio buttons.
+ */
   @Input()
   public icon: string;
 
+/**
+ * Specifies a type to set the background color after users select a radio button where the icon
+ * property displays an icon in place of the radio button. The valid options correspond
+ * [the label component's](https://developer.blackbaud.com/skyux/components/label) label types.
+ * `info` creates a blue background, `success` creates a green background, `warning` creates an
+ * orange background, and `danger` creates a red background.
```

Comment:
We usually list options in alphabetical order, so `danger` should really be first in the list.

Reply:
Looks like this wasn't addressed. Not a huge deal, but just pointing it out so that we can either address it or resolve this comment thread as "will not address."

Reply:
Done